webpack.config.js

```diff
@@ -0,0 +1,32 @@
+// module.exports = {
+//   resolve: {
+//     modules: ['node_modules'],
+//     fallback: {
+//       fs: true,
+//       tls: false,
+//       net: false,
+//       path: false,
+//       zlib: false,
+//       http: false,
+//       https: false,
+//       stream: false,
+//       crypto: false,
+//       'crypto-browserify': require.resolve('crypto-browserify'),
+//       'path-browserify': { path: require.resolve('path-browserify') },
+//     },
+//   },
+//   // entry: [...],
+//   // output: {...},
+//   // module: {
+//   //   rules: [...]
+//   // },
+//   // plugins: [...],
+//   // optimization: {
+//   //   minimizer: [...],
+//   // },
+//   // node: {
+//   //   fs: 'empty',
+//   //   net: 'empty',
+//   //   tls: 'empty'
+//   // },
+// };
```

Comment:
- [ ] (Required) You can delete this file if it is of no use to you. Kindly implement it. 👍